### DISCUSSION TITLE
github api

### DISCUSSION THREAD

**Discussion by BidemiEnoch** [Tekipeps/aul-forum#67] [Ideas]
So I was working with the github api to fetch data about this repo for the about page(contribution data), now i realized doing `axios.get(githubapi)` on the frontend is a no-go since the api has a rate limit(Getting an authentication increases this rate limit but still... the rate limit would be imposed on the user of the app(through their IP address)).
So if you could just fetch data from the api to the backend say... every 1 hour? and cache it some db or something and integrate into aul forum's api, so i could fetch data on the frontend, this way, we can never worry about a rate limit. 
**required endpoints**
- https://api.github.com/repos/Tekipeps/aul-forum
- https://api.github.com/repos/Tekipeps/aul-forum/contributors
- https://api.github.com/repos/Tekipeps/aul-forum/languages

**Comment by Tekipeps** [comment]
I will look into this soon...  fetching from the backend is a long way
we can fetch and persist it in browser cache then  subsequent request will used the cached data

**Reply by BidemiEnoch** [reply]
Yh, no p, since thats frontend stuff, ill do it myself. brilliant idea
